ts/Tabs/index.tsx

```diff
@@ -0,0 +1,241 @@
+import {MDCTabBar, MDCTabBarScroller} from '@material/tabs';
+
+import {h} from 'preact';
+import {MDCComponent, MDCFoundation} from '../../MaterialComponentsWeb';
+import MaterialComponent from '../MaterialComponent';
+
+declare class MDCTabFoundation extends MDCFoundation<MDCTab> {
```

Comment:
I am kinda leaning against declaring the encapsulated classes.

Reply:
As already said, that's the one con, but someone implement them before I could, else the definitions hadn't worked